ledger/ledger-on-memory/src/main/scala/com/daml/ledger/on/memory/package.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.on
+
+import com.daml.ledger.participant.state.kvutils.api.{LedgerReader, LedgerWriter}
+import com.daml.platform.akkastreams.dispatcher.Dispatcher
+import com.daml.resources.ResourceOwner
+
+package object memory {
+  type Index = Int
+
+  type KeyValueLedger = LedgerReader with LedgerWriter
```

Comment:
This probably belongs in _kvutils_. We already define it in _kvutils/app_'s package object; perhaps it can be moved.

Reply:
Moved it to `com.daml..ledger.participant.state.kvutils.api`.